### PR TITLE
bug 1712579 no pull secret for RHEL playbook

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1712579

built draft here: http://file.rdu.redhat.com/kalexand/052819/BZ1712579/machine_management/adding-rhel-compute.html

@mtnbikenc, I know your PR hasn't merged yet, but does this change look right to you?

### CODE REVIEW

File: modules/rhel-adding-node.adoc [hunk 1]
```diff
@@ -47,8 +38,6 @@ machines.
 <2> If you do not specify `root` for the `ansible_user`, you must set `ansible_become`
 to `True` and assign the user sudo permissions.
 <3> Specify the path to the `kubeconfig` file for your cluster.
-<4> Specify the path to the file that contains the pull secret for the image
-registry for your cluster.
 <5> List each RHEL machine to add to your cluster. You must provide the
```
Comment (mtnbikenc) [inline modules/rhel-adding-node.adoc]: Nit: Update to `<4>`

**Review by mtnbikenc — COMMENTED** [review]
Overall looks good.  One minor nit that doesn't affect rendering.

File: modules/rhel-adding-node.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/rhel-adding-node.adoc]: Thanks!

**Comment by vikram-redhat** [comment]
@kalexand-rh Once 4.1 GAs, change the milestone from future release to next release and you may need to tag with the enterprise-4.2 label.

**Comment by kalexand-rh** [comment]
@mtnbikenc cherrypicked this change to 4.1, so it will be picked up in a .z release.

**Comment by kalexand-rh** [comment]
@mtnbikenc said that he actually wasn't going to cherry-pick this back to 4.1, so it's only applicable to 4.2. I've addressed the last issue that @wmengRH mentioned on the source bug.

**Pull request merged**